redhat-cop/ah_configuration
# Add ah_ee_registry_sync module

*Merge #77 first*

### What does this PR do?
Adds ah_ee_registry_sync module to the collection. It kicks off the task to sync all the container repos. Each registry has X container repos so this kicks off a task which has X child tasks. This tracks those and reports back when its all done (if wait set to true)

### How should this be tested?
Not the easist thing to do but something like:

```
 - name: Ensure the registry description and README are set
      redhat_cop.ah_configuration.ah_ee_registry:
        name: rrio
        username: XXX
        password: YYY
        url: https://registry.redhat.io
        ah_hostname: https://localhost:8443
        ah_username: admin
        ah_password: password
        validate_certs: false

    - name: Sync the registry
      redhat_cop.ah_configuration.ah_ee_registry_sync:
        name: rrio
        interval: 5
        ah_hostname: https://localhost:8443
        ah_username: admin
        ah_password: password
        validate_certs: false
```
works (though it takes FOREVER for the syncs to happen - though that may be because my AH is very under-specced)

### Is there a relevant Issue open for this?
resolves #73 

### Other Relevant info, PRs, etc.
Merge #77 first as this followed on because I needed some of the objects
